### ISSUE TITLE
A question about the details of network training

### ISSUE THREAD

**Issue by XiaxiaBubble** [RazvanDu/DUCK-Net#3]
Thank you for your wonderful work!
I was very impressed with your article, the network you proposed is concise and powerful. 
I have a question about the details of network training: Could you please tell me that during the training phase, did you train on these datasets separately, or combined them into one large training set?
Your response would be highly appreciated, and thank you again for your great work!

**Comment by RazvanDu** [comment]
Hey, thanks for your feedback!

We trained and tested the network on each data set independently, besides the generalization studies where we trained on one data set and tested on another. 

I hope that clarifies the issue.

**Comment by XiaxiaBubble** [comment]
Ok, thanks for the reply! :)

**Comment by garlicman-man** [comment]
hi, in the paper I see "Segmentation accuracy (Dice coefcient, Jaccard index, Accuracy, Recall and Precision) on the
Kvasir-SEG dataset, the models being trained on the CVC-ClinicDB dataset. Best model results are in bold." 

This is different from the one in the ParaNet paper, have you tried that kind of data?

**Comment by maenstru56** [comment]
Hey, can you please provide a link to the paper you're reffering to and the pages where the experiment/data you're reffering to is described?

**Comment by garlicman-man** [comment]
https://paperswithcode.com/paper/pranet-parallel-reverse-attention-network-for

**Comment by maenstru56** [comment]
We did not try the kind of generalization testing techniques presented in the PraNet paper (by combining certain datasets), we only did 3 kinds of tests:
- Training and testing the algorithm on each dataset individually 
- Training the algorithm on the Kvasir-SEG dataset and testing it on the CVC-ClinicDB dataset (testing for generalization capabilities)
- Training the algorithm on the CVC-ClinicDB dataset and testing it on the Kvasir-SEG dataset (testing for generalization capabilities)
